ocaml/xapi/xapi_globs.ml

```diff
@@ -927,7 +927,7 @@ let winbind_debug_level = ref 2
 
 let winbind_cache_time = ref 60
 
-let winbind_machine_pwd_timeout = ref (7 * 24 * 3600)
+let winbind_machine_pwd_timeout = ref (7. *. 24. *. 3600.)
```

Comment:
It might be best to handle this on a systemd timer, so you can also set the time. E.g. you may want to run this rotation during a maintenance window (see above: password rotations risk introducing a window where password authentications would fail, and you don't want that in the middle of your working day).

Reply:
windows DC keep latest 2 version of password, 
so password rotating does not break the authentication